src/views/LoginView.vue

```diff
@@ -1,114 +1,196 @@
-    </v-row>
-
-    <v-row class="mt-0">
-      <v-col :cols="$vuetify.display.mdAndUp ? 6 : 12" class="d-flex justify-start flex-column mt-4">
-        <blockquote class="quote">
+        </div>    
+      </v-container>
+    </section>
+
+    <!--////////////////////////// 
+     /////Login Section /////// 
+    //////////////////////////-->
+
+    <section>
+      <v-container id="login-container" class="container">
+        <img id="logo" alt="Cuttle logo" src="/img/logo.png" />
+        <v-row>
+          <!-- Left side form -->
+          <v-col id="username-login-form" :cols="$vuetify.display.mdAndUp ? 8 : 12" >
+            <p class="text-h4 text-center font-weight-medium mb-10 ">
+              {{ formHeaderText }}
+            </p>
+
+            <v-form
+              v-model="isFormValid"
+              id="login"
+              @submit.prevent="submitLogin"
+              class="mx-auto"
+            >
+              <label for="username" class="text-body-1 font-weight-bold " >Username</label>
+              <v-text-field
+                ref="usernameInput"
+                id="username"
+                class="mt-4"
+                v-model="username"
+                variant="solo"
+                :rules="usernameRules"
+                :dense="$vuetify.display.mdAndDown ? true : false"
+                data-cy="username"
+                autocomplete="username"
+              />
+
+              <label for="password" class="text-body-1 font-weight-bold">Password</label>
+              <v-text-field
+                id="password"
+                class="my-4"
+                v-model="pw"
+                variant="solo"
+                :rules="passwordRules"
+                :dense="$vuetify.display.mdAndDown ? true : false"
+                :type="showPass ? 'text' : 'password'"
+                data-cy="password"
+                :autocomplete="isLoggingIn ? 'current-password' : 'new-password'"
+                :append-inner-icon="showPass ? 'mdi-eye' : 'mdi-eye-off'"
+                @click:append-inner="showPass = !showPass"
+              />
+         
+              <div
+                id="login-button-container"
+                class="d-flex flex-column flex-md-row
+             justify-space-between align-center flex-wrap"
+              >
+                <v-btn
+                  class="px-16"
+                  :loading="loading"
+                  :disabled="!isFormValid"
+                  :color=" isFormValid ? 'newPrimary' : 'disabled'"
+                  type="submit"
+                  size="x-large"
+                  text-color="white"
+                  data-cy="submit"
+                >
+                  {{ buttonText }}
+                </v-btn>
+                <v-btn
+                  class="px-0 text-h6"
+                  color="black"
+                  variant="text"
+                  data-cy="switch-mode"
+                  @click="switchMode"
```

Comment:
Now that we have separate pages for `LoginView` and `SignupView`, this function can be made simpler i.e. it only navigates to `SignupView`.

OR we could pull the guts of the form into a shared `AuthenticationForm.vue` component that does both and then we could perhaps keep `switchMode` to handle both cases. @Haviles04 penny for your thoughts

Reply:
nvm it's already one page I'm silly